database/seeders/ShopSeeder.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Shop;
+use Illuminate\Database\Seeder;
+
+class ShopSeeder extends Seeder
+{
+    /**
+     * Seed the application's database.
+     * itemsテーブルに対して1つしか紐づかない前提
+     * @return void
+     */
+    public function run()
+    {
+        Shop::factory(10)->create([
+            'user_id' => 1,
```

Comment:
ShopFactoryでは、user_id => 1 と指定されてるのでこの記述は不要かと思います。
```suggestion
        Shop::factory(10)->create();
```